packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -281,11 +289,24 @@ function Thread( {
 					className="editor-collab-sidebar-panel__child-thread"
 					spacing="2"
 				>
+					<Button
+						className="editor-collab-sidebar-panel__skip-link"
+						variant="link"
+						__next40pxDefaultSize
+						onClick={ () => {
+							replyTextareaRef.current?.focus();
+						} }
```

Comment:
Let's avoid this approach for now. Once #71814 is merged, it will be easier to focus textarea.

Reply:
Yes 👍